backend/supabase/seed.sql

```diff
@@ -471,18 +471,17 @@ create index if not exists contract_bets_data_gin on contract_bets using GIN (da
 /* serves bets API pagination */
 create index if not exists contract_bets_bet_id on contract_bets (bet_id);
 
-/* serving stats page, recent bets API */
-create index if not exists contract_bets_created_time_global on contract_bets (created_time desc);
-
 /* serving activity feed bets list */
 create index if not exists contract_bets_activity_feed on contract_bets (is_ante, is_redemption, created_time desc);
 
+/* serving update contract metrics */
+create index if not exists contract_bets_historical_probs
+  on contract_bets (created_time)
+  include (contract_id, prob_before, prob_after);
```

Comment:
How is this include thing different than just making a normal index like:
`contract_bets (created_time, contract_id, prob_before, prob_after)`?

Reply:
Oh I see you just posted about this in discord: https://discord.com/channels/915138780216823849/1108516565676200018/1111042390510272602